src/components/BreakoutRoomsEditor/SendMessageDialog.vue

```diff
@@ -4,40 +4,40 @@
 -->
 
 <template>
-	<NcModal ref="modal"
+	<NcDialog ref="modal"
+		:name="dialogTitle"
 		:container="container"
-		v-on="$listeners">
-		<div class="send-message-dialog">
-			<h2 class="send-message-dialog__title">
-				{{ dialogTitle }}
-			</h2>
-			<NewMessage v-if="modalContainerId"
-				:key="modalContainerId"
-				ref="newMessage"
-				role="region"
-				:token="token"
-				:container="modalContainerId"
-				:aria-label="t('spreed', 'Post message')"
-				:broadcast="broadcast"
-				@sent="handleMessageSent"
-				@failure="handleMessageFailure" />
-		</div>
-	</NcModal>
+		close-on-click-outside
+		size="normal"
+		v-on="$listeners"
+		@update:open="$emit('close')">
```

Comment:
isn't it more accurate to use `@closing` event here ?

Reply:
From nextcloud/vue - I think we should use the one after transition ends:

```
		/**
		 * Handle closing the dialog, optional out transition did not run yet
		 */
		const handleClosing = () => {
			showModal.value = false
			/**
			 * Emitted when the dialog is closing, so the out transition did not finish yet
			 */
			emit('closing')
		}

		/**
		 * Handle dialog closed (out transition finished)
		 */
		const handleClosed = () => {
			showModal.value = true
			/**
			 * Emitted then the dialog is fully closed and the out transition run
			 */
			emit('update:open', false)
		}
```

Reply:
Alright, because `@update:open` is not only restricted to emit `false` value (for now, yes but it might change).